### PR TITLE
doc: add minutes for July 21 2016

### PR DESCRIPTION
**Pull request by mhdawson**
Adding minutes

### CODE REVIEW

File: meetings/2016-07-18/minutes.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+# Present
+
+* Yunong Xiao @yunongx
+* Dave Pacheco @dapsays
+* Joshua M. Clulow @jmclulow
+* Michael Dawson  @mhdawson
+* Richard Chamberlain @rnchamberlain
+* Ali Sheik @ofrobots
+* Luca
+
```
Comment (williamkapke) [inline meetings/2016-07-18/minutes.md:13]: Ali and Luca do not seem to have made it to this meeting.

Also, just a personal nit: on GitHub, I assume @ -mentinos are for GitHub- not twitter. (a few above are not the same between the 2 platforms)

**Comment by williamkapke** [comment]
Looks correct to me with the exception of the nit(s) mentioned.

**Comment by mhdawson** [comment]
@williamkapke thanks for catching those, pushed commit with updates.  

@nodejs/post-mortem can I get an l.g.t.m from one of the attendees ?

**Comment by rnchamberlain** [comment]
Thanks @mhdawson, LGTM.

**Comment by mhdawson** [comment]
Landed as 9ffd2f03678391790914d314a15d2467fe751d66

**Pull request closed without merging**